albumentations/augmentations/bbox_utils.py

```diff
@@ -1,10 +1,88 @@
 from __future__ import division
+from albumentations.core.utils import Params, DataProcessor
 
 import numpy as np
 
 __all__ = ['normalize_bbox', 'denormalize_bbox', 'normalize_bboxes', 'denormalize_bboxes', 'calculate_bbox_area',
            'filter_bboxes_by_visibility', 'convert_bbox_to_albumentations', 'convert_bbox_from_albumentations',
-           'convert_bboxes_to_albumentations', 'convert_bboxes_from_albumentations']
+           'convert_bboxes_to_albumentations', 'convert_bboxes_from_albumentations', 'BboxParams']
+
+
+class BboxParams(Params):
+    def __init__(self, format=None, label_fields=None, min_area=0.0, min_visibility=0.0):
```

Comment:
I'd suggest to not use `format=None` to disable boxes transform. I'd rather say if `bbox_params is None` then it's disabled.